doc/performance-investigations/0022-upgrade-cloud-run-execution-environment-gen2/summary.md

```diff
@@ -0,0 +1,115 @@
+|------------------------------------------|------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| **GET** <br/>(*/submitted/download-pdf*) | 1900ms                                                                             | 1600ms                                                                                  |
+| Total Requests                           | 2,176                                                                              | 2,146                                                                                   |
+| 99th percentile Max CPU Usage (%)        | 30.9%                                                                              | 29.9%                                                                                   |
+| Total Failures                           | 0                                                                                  | 0                                                                                       |
+| Error Percentage                         | 0.0%                                                                               | 0.0%                                                                                    |
+| Results                                  | [benchmark_results](outputs/baseline/2023-02-27/Results_For_Latest_Benchmark.yaml) | [benchmark_results](outputs/investigation/2023-02-27/Results_For_Latest_Benchmark.yaml) |
+| Performance Graph                        | [result](outputs/baseline/2023-02-27/performance_graph.png)                        | [result](outputs/investigation/2023-02-27/performance_graph.png)                        |
+
+#### Detailed Results.
+
+| Execution Environment | Type | Name                    | Request Count | Failure Count | Average Response Time | Average Content Size | Requests/s | 50%  | 66%  | 75%  | 80%  | 90%  | 95%  | 98%  | 99%  | 99.90% | 99.99% | 100% | outputs                                     |
+|-----------------------|------|-------------------------|---------------|---------------|-----------------------|----------------------|------------|------|------|------|------|------|------|------|------|--------|--------|------|---------------------------------------------|
+| Gen1                  | GET  | /submitted/download-pdf | 2176          | 0             | 1134.83901            | 352743.324           | 1.81477554 | 1100 | 1100 | 1200 | 1200 | 1200 | 1300 | 1600 | 1900 | 2300   | 2700   | 2700 | [outputs](outputs/baseline/2023-02-27)      |
+| Gen2                  | GET  | /submitted/download-pdf | 2146          | 0             | 1028.35708            | 353894.429           | 1.78933408 | 1000 | 1000 | 1000 | 1100 | 1100 | 1100 | 1300 | 1600 | 2400   | 2700   | 2700 | [outputs](outputs/investigation/2023-02-27) |
```

Comment:
Was the Gen2 investigation benchmark only run once? If not might have been worth running a couple of times just in case that particular run was an outlier - this is something we have normally done on previous investigations

Reply:
As per google doc - Gen2 works better with consistent load, hence thought its better to run on a fresh single run. I have ran the tests again and added the findings.